erts/emulator/hipe/hipe_amd64.c

```diff
@@ -96,12 +102,70 @@ static void *alloc_code(unsigned int alloc_bytes)
     return erts_alloc(ERTS_ALC_T_HIPE_EXEC, alloc_bytes);
 }
 
+static int check_callees(Eterm callees)
+{
+    Eterm *tuple;
+    Uint arity;
+    Uint i;
+
+    if (is_not_tuple(callees))
+	return -1;
```

Comment:
check_callees() must accept NIL meaning no callees and return 0.

Reply:
@sverker I copied this from `hipe_arm.c` which does not check if it is NIL either.

We got a NIL here, because `lib/kernel/src/hipe_unified_loader.erl` has been changed but bootstrap beam files are left not updated.